src/platform/dnne.h

```diff
@@ -75,6 +75,8 @@ enum failure_type
 {
     failure_load_runtime = 1,
     failure_load_export,
+    failure_get_current_app_domain_id,
+	failure_get_current_app_domain_id_backup_try
```

Comment:
```suggestion
```

This level of clarity isn't all that helpful since the distinction doesn't change what needs to be debugged. Appreciate the thoughtfulness though.